deploy-board/deploy_board/templates/clusters/cluster_configuration.html

```diff
@@ -418,6 +418,7 @@ <h4 class="panel-title pull-left pointer-cursor">
     var currentCluster = info.currentCluster;
     if (currentCluster!=null){
         var currentPlacements = currentCluster.placement.split(',');
+        var currentPlacementObjs = info.placements.filter(function(placement) {return currentPlacements.includes(placement.id)});
```

Comment:
This doesn't seem to handle placement changes. 
I'd recommend handle the changes in 
```
                $(".chosen-select").chosen({
                        "width": "100%"
                    })
                    .change(function() {
                        capacitySetting.selectedPlacements = $('.chosen-select').val()
                    });
```
Besides, I think the calculation logic can move to `getDefaultPlacement()`, similar to the implementation of `getSimpleList()`.

Reply:
To clarify: 
-You recommend moving the calculation to `getDefaultPlacement()`
-Handle changes for remaining capacity in the place changes for selected placements are made

Reply:
Yes.